staging/src/k8s.io/api/flowcontrol/v1alpha1/types.go

```diff
@@ -28,7 +28,8 @@ const (
 	NonResourceAll = "*"
 	NameAll        = "*"
 
-	NamespaceEvery = "*" // matches every particular namespace
+	NamespaceEvery    = "*" // matches every particular namespace
+	ResourceNameEvery = "*" // matches every particular resource name
```

Comment:
as noted, I'd prefer if we could make this 100% a promotion and add the other field in a followup.